substrate/frame/child-bounties/src/lib.rs

```diff
@@ -276,15 +296,18 @@ pub mod pallet {
 			)?;
 
 			// Get child-bounty ID.
-			let child_bounty_id = ChildBountyCount::<T>::get();
+			let child_bounty_id = ParentTotalChildBounties::<T>::get(parent_bounty_id);
 			let child_bounty_account = Self::child_bounty_account_id(child_bounty_id);
 
 			// Transfer funds from parent bounty to child-bounty.
 			T::Currency::transfer(&parent_bounty_account, &child_bounty_account, value, KeepAlive)?;
```

Comment:
we forgot about child bounty account. it changes and the balances should be transferred to the new account. the account should be derived from (parent_id, child_id). please run try runtime to make sure the transfers do not fail and move all the balance

Reply:
Transferred everything to new child bounty accounts derived by parent id and child id